diffrax/brownian/base.py

```diff
@@ -1,14 +1,49 @@
 import abc
+from typing import Optional, Union
 
-from ..custom_types import Array, PyTree, Scalar
+import equinox as eqx
+import jax
+from jax import tree_util as jtu
+
+from ..custom_types import Array, LevyVal, PyTree, Scalar
 from ..path import AbstractPath
 
 
+def _levy_tree_transpose(tree_shape, spacetime_levyarea, tree):
+    """Helper that takes a PyTree of LevyVals and transposes
+    into a LevyVal of PyTrees.
+
+    Args:
+        tree_shape:
+        spacetime_levyarea:
+        tree:
+
+    Returns:
+
```

Comment:
Can you fill in the docstring here? (And using the same style as the rest of Diffrax, i.e. markdown, rather than the Google-style stub currently here.)